cfbs/commands.py

```diff
@@ -133,22 +134,27 @@ def init_command(index=None, non_interactive=False) -> int:
     if index:
         config["index"] = index
 
+    do_git = get_args().git
```

Comment:
Here and below you are effectively introducing another global, another "hidden dependency" for this function. I don't like this - IMO these things should be a part of the config, `CFBSConfig` is the current config of the appliation, meaning JSON + command line options. I think it would be useful if we follow some patterns here, for example that the calling a `_command` functions with the right arguments is equivalent to running the cfbs command.

I can do some refactoring and add some comments to make this more clear.